pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>com.okta</groupId>
         <artifactId>okta-parent</artifactId>
-        <version>4</version>
+        <version>5-SNAPSHOT</version>
```

Comment:
Looks like modifying the version is causing your tests to fail. Can you double check?

Reply:
that is the reason, the SNAPSHOT isn't publish, so this is just queued up until that parent is released